src/assets/mixins.styl

```diff
@@ -1,16 +1,29 @@
 //fonts mixins
-play($weight)
-  font-family 'Playfair Display', serif
-  font-weight $weight
-
-
-source($weight)
-  font-family 'Source Sans Pro', sans-serif
+setFont($family, $weight)
+  if $family == 'playfair'
+    font-family 'Playfair Display', serif
+  if $family == 'source'
+    font-family 'Source Sans Pro', sans-serif
```

Comment:
variable

Reply:
Because this is a mixin I don't see why this should be a variable

Reply:
is being used in main.styl aswell so either you only use it here and everytime you use the font you use the mixin or change it to a variable.